### PR TITLE
Label 2500+ images

### PR DESCRIPTION
**Pull request by amaaniqbal**
Labelled 2562 images and updated line 218 in autowebcompat/utils file to add extra argument newline='' while opening csv file for writing in order to fix printed extra line gap in windows.

### CODE REVIEW

File: autowebcompat/utils.py [hunk 1]
```diff
@@ -1,295 +1,295 @@
-import csv
```
[the remark this answers is not in the document]
Comment (amaaniqbal) [reply autowebcompat/utils.py]: Okay then let me remove that file completely

**Comment by marco-c** [comment]
It looks like your labels are not sorted, could you:
1) Rebase on top of master;
2) Label one more case, so that the labels get sorted;
3) Run generate_labels.py

**Comment by amaaniqbal** [comment]
git rebase master is returning everything up to date

What do you mean by label one more case?

**Comment by marco-c** [comment]
> git rebase master is returning everything up to date

You need to run:
`git pull upstream master --rebase`

> What do you mean by label one more case?

Label just another couple of screenshots, so the label.py script will automatically sort them in the CSV file.

**Comment by amaaniqbal** [comment]
Should I run the `git pull upstream master --rebase` command on the current branch or master.

**Comment by sagarvijaygupta** [comment]
@amaaniqbal  In your current branch

File: autowebcompat/utils.py [hunk 2]
```diff
@@ -215,7 +215,7 @@ def to_categorical_label(label, classification_type):
 
 
 def write_labels(labels, file_name='labels.csv'):
-    with open(file_name, 'w') as f:
+    with open(file_name, 'w', newline='') as f:
```
Comment (marco-c) [inline autowebcompat/utils.py]: Can you remove this change, since it's handled by your other PR?

**Pull request merged**